### ISSUE TITLE
Support for AAC 7.1?

### ISSUE THREAD

**Issue by THX723** [darknebular/Wrapper_VideoStation#52]
Would it be possible to have an option to transcode to AAC 7.1, instead of the default AAC 5.1?

**Comment by THX723** [comment]
Yes, I do agree. Still would be really cool to allow a 3rd 'expert' option for those with higher powered CPUs. I've got a collection of movies encoded as 7.1 TrueHD.

Thanks for your development work btw.

**Comment by darknebular** [comment]
If I get some free time, I will look into creating an option to leave the audio codecs intact, I think it would be good this way due to CPU resource issues and you can have 7.1, the problem is if your player does not support that format.

**Comment by darknebular** [comment]
Finally I will not add the support for 7.1.
